Example/PaymentSheet Example/PaymentSheet Example/PlaygroundController.swift

```diff
@@ -0,0 +1,603 @@
+        }
+        alertController.addAction(OKAction)
+        return alertController
+    }
+
+    private var subscribers: Set<AnyCancellable> = []
+
+    init(settings: PaymentSheetTestPlaygroundSettings) {
+        // Enable experimental payment methods.
+        //        PaymentSheet.supportedPaymentMethods += [.link]
+        PaymentSheet.enableACHV2InDeferredFlow = true // TODO(https://jira.corp.stripe.com/browse/BANKCON-6731) Remove this.
+
+        // Hack to ensure we don't force the native flow unless we're in a UI test
+        if ProcessInfo.processInfo.environment["UITesting"] == nil {
+            UserDefaults.standard.removeObject(forKey: "FINANCIAL_CONNECTIONS_EXAMPLE_APP_ENABLE_NATIVE")
+        } else {
+            // This makes the Financial Connections SDK use the native UI instead of webview. Native is much easier to test.
+            UserDefaults.standard.set(true, forKey: "FINANCIAL_CONNECTIONS_EXAMPLE_APP_ENABLE_NATIVE")
+        }
+        self.settings = settings
+
+        $settings.sink { _ in
+            self.load()
+        }.store(in: &subscribers)
+    }
+
+    func buildPaymentSheet() {
+        let mc: PaymentSheet
+
+        switch self.settings.integrationType {
+        case .normal:
+            switch self.settings.mode {
+            case .payment, .paymentWithSetup:
+                mc = PaymentSheet(paymentIntentClientSecret: self.clientSecret!, configuration: configuration)
+            case .setup:
+                mc = PaymentSheet(setupIntentClientSecret: self.clientSecret!, configuration: configuration)
+            }
+        case .deferred_csc, .deferred_ssc, .deferred_mp, .deferred_mc:
+            mc = PaymentSheet(intentConfiguration: intentConfig, configuration: configuration)
+        }
+
+        self.paymentSheet = mc
+    }
+
+        func didTapShippingAddressButton() {
+            // Hack, should do this in SwiftUI
+            let rvc = UIApplication.shared.windows.first!.rootViewController!
+            rvc.present(UINavigationController(rootViewController: addressViewController!), animated: true)
+        }
+
+        func didTapEndpointConfiguration() {
+            // Hack, should do this in SwiftUI
+            let rvc = UIApplication.shared.windows.first!.rootViewController!
+            let endpointSelector = EndpointSelectorViewController(delegate: self,
+                                                                  endpointSelectorEndpoint: PaymentSheetTestPlaygroundSettings.endpointSelectorEndpoint,
+                                                                  currentCheckoutEndpoint: checkoutEndpoint)
+            let navController = UINavigationController(rootViewController: endpointSelector)
+            rvc.present(navController, animated: true, completion: nil)
+        }
+
+        func didTapResetConfig() {
+            self.settings = PaymentSheetTestPlaygroundSettings.defaultValues()
+            self.load()
```

Comment:
If we update settings, i woudldn't expect to need an explicit call to load() here.